### PR TITLE
feat: Refactor Enable And Disable Module Commands

### LINKED ISSUE #1806: Change kyma enable / disable modue command to add / delete
**Description**

There is a difference in the name of the action performed by a user who wants to start using a module. 

For those who use Kyma CLI the action is **enable/disable**, but in Kyma Dashboard they **Add** or **Delete** a module.

The difference may be misleading for users who read about the actions in the documentation.

Unification would require changing the verbs used in the commands.

Confirm with @pbochynski and @valentinvieriu if the **Add/Delete** is the final version of the action performed by a user in UI.

**Reasons**

Unified naming convention and better user experience.

**ACs**
- [x] Change naming of `enable` and  `disable` to `add` and `delete`
- [x] Update documentation in all relevant places (Pair with @mmitoraj )
- [ ] Announce to other teams
- [x] Change implementation into pipeline
- [x] Give "new" commands an alias to old naming, to have a proper transition period (check if this exists in cobra)

**Attachments**

### PR DESCRIPTION
**Pull request by LeelaChacha**
**Description**

Changes proposed in this pull request:

- Rename `kyma alpha enable module` and `kyma alpha disable module` commands
- Update docs

**Related issue(s)**
Closes #1806

### CODE REVIEW

File: cmd/kyma/alpha/add/module/module.go [hunk 1]
```diff
@@ -31,8 +31,8 @@ func NewCmd(o *Options) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "module [name] [flags]",
-		Short: "Enables a module in the cluster or in the given Kyma resource.",
-		Long: `Use this command to enable Kyma modules available in the cluster.
+		Short: "Activates a module in the cluster or in the given Kyma resource.",
+		Long: `Use this command to activate Kyma modules available in the cluster.
```
Comment (jeremyharisch) [inline cmd/kyma/alpha/add/module/module.go:35]: I would stick to `add`

```suggestion
		Short: "Add a module in the cluster or in the given Kyma resource.",
		Long: `Use this command to add Kyma modules available in the cluster.
```

File: cmd/kyma/alpha/delete/module/module.go [hunk 2]
```diff
@@ -29,8 +29,8 @@ func NewCmd(o *Options) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "module [name] [flags]",
-		Short: "Disables a module in the cluster or in the given Kyma resource.",
-		Long: `Use this command to disable active Kyma modules in the cluster.
+		Short: "Deactivates a module in the cluster or in the given Kyma resource.",
+		Long: `Use this command to deactivate Kyma modules in the cluster.
```
Comment (jeremyharisch) [inline cmd/kyma/alpha/delete/module/module.go:33]:
```suggestion
		Short: "Delete a module in the cluster or in the given Kyma resource.",
		Long: `Use this command to delete Kyma modules in the cluster.
```

File: cmd/kyma/alpha/add/add.go [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
 	"github.com/spf13/cobra"
 )
 
 func NewCmd(o *cli.Options) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "enable",
-		Short: "Enables a resource in the Kyma cluster.",
+		Use:     "add",
+		Aliases: []string{"enable"},
+		Short:   "Enables a resource in the Kyma cluster.",
```
Comment (mmitoraj) [inline cmd/kyma/alpha/add/add.go:13]:
```suggestion
		Short:   "Adds a resource to the Kyma cluster.",
```

File: cmd/kyma/alpha/add/module/module.go [hunk 4]
```diff
@@ -31,8 +31,8 @@ func NewCmd(o *Options) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "module [name] [flags]",
-		Short: "Enables a module in the cluster or in the given Kyma resource.",
-		Long: `Use this command to enable Kyma modules available in the cluster.
+		Short: "Add a module in the cluster or in the given Kyma resource.",
```
Comment (mmitoraj) [inline cmd/kyma/alpha/add/module/module.go:34]:
```suggestion
		Short: "Adds a module in the cluster or in the given Kyma resource.",
```

File: cmd/kyma/alpha/delete/delete.go [hunk 5]
```diff
@@ -1,15 +1,16 @@
...
 	"github.com/spf13/cobra"
 )
 
 func NewCmd(o *cli.Options) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "disable",
-		Short: "Disables a resource in the Kyma cluster.",
+		Use:     "delete",
+		Aliases: []string{"disable"},
+		Short:   "Disables a resource in the Kyma cluster.",
```
Comment (mmitoraj) [inline cmd/kyma/alpha/delete/delete.go:13]:
```suggestion
		Short:   "Deletes a resource from the Kyma cluster.",
```

File: cmd/kyma/alpha/delete/module/module.go [hunk 6]
```diff
@@ -29,8 +29,8 @@ func NewCmd(o *Options) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "module [name] [flags]",
-		Short: "Disables a module in the cluster or in the given Kyma resource.",
-		Long: `Use this command to disable active Kyma modules in the cluster.
+		Short: "Delete a module in the cluster or in the given Kyma resource.",
```
Comment (mmitoraj) [inline cmd/kyma/alpha/delete/module/module.go:32]:
```suggestion
		Short: "Deletes a module from the cluster or the given Kyma resource.",
```

File: cmd/kyma/alpha/delete/module/module.go [hunk 7]
```diff
@@ -29,8 +29,8 @@ func NewCmd(o *Options) *cobra.Command {
...
+		Long: `Use this command to delete Kyma modules in the cluster.
```
Comment (mmitoraj) [inline cmd/kyma/alpha/delete/module/module.go:33]:
```suggestion
		Long: `Use this command to delete Kyma modules from the cluster.
```

File: docs/gen-docs/kyma_alpha.md [hunk 8]
```diff
@@ -22,10 +22,10 @@ Alpha commands are experimental, unreleased features that should only be used by
 ## See also
 
 * [kyma](kyma.md)	 - Controls a Kyma cluster.
+* [kyma alpha add](kyma_alpha_add.md)	 - Enables a resource in the Kyma cluster.
```
Comment (mmitoraj) [inline docs/gen-docs/kyma_alpha.md:25]:
```suggestion
* [kyma alpha add](kyma_alpha_add.md)	 - Adds a resource to the Kyma cluster.
```
I believe

File: docs/gen-docs/kyma_alpha.md [hunk 9]
```diff
@@ -22,10 +22,10 @@ Alpha commands are experimental, unreleased features that should only be used by
...
 * [kyma alpha create](kyma_alpha_create.md)	 - Creates resources on the Kyma cluster.
+* [kyma alpha delete](kyma_alpha_delete.md)	 - Disables a resource in the Kyma cluster.
```
Comment (mmitoraj) [inline docs/gen-docs/kyma_alpha.md:27]:
```suggestion
* [kyma alpha delete](kyma_alpha_delete.md)	 - Deletes a resource from the Kyma cluster.
```

File: docs/gen-docs/kyma_alpha_add.md [hunk 10]
```diff
@@ -22,5 +22,5 @@ Use this command to enable a resource in the Kyma cluster.
 ## See also
 
 * [kyma alpha](kyma_alpha.md)	 - Experimental commands
-* [kyma alpha enable module](kyma_alpha_enable_module.md)	 - Enables a module in the cluster or in the given Kyma resource.
+* [kyma alpha add module](kyma_alpha_add_module.md)	 - Add a module in the cluster or in the given Kyma resource.
```
Comment (mmitoraj) [inline docs/gen-docs/kyma_alpha_add.md:25]:
```suggestion
* [kyma alpha add module](kyma_alpha_add_module.md)	 - Adds a module to the cluster or the given Kyma resource.
```

File: docs/gen-docs/kyma_alpha_add_module.md [hunk 11]
```diff
@@ -1,12 +1,12 @@
 ---
-title: kyma alpha enable module
+title: kyma alpha add module
 ---
 
-Enables a module in the cluster or in the given Kyma resource.
+Add a module in the cluster or in the given Kyma resource.
```
Comment (mmitoraj) [inline docs/gen-docs/kyma_alpha_add_module.md:5]:
```suggestion
Adds a module to the cluster or the given Kyma resource.
```

File: docs/gen-docs/kyma_alpha_add_module.md [hunk 12]
```diff
@@ -53,5 +53,5 @@ Enable "my-module" from "alpha" channel in "default-kyma" in "kyma-system" Names
 
 ## See also
 
-* [kyma alpha enable](kyma_alpha_enable.md)	 - Enables a resource in the Kyma cluster.
+* [kyma alpha add](kyma_alpha_add.md)	 - Enables a resource in the Kyma cluster.
```
Comment (mmitoraj) [inline docs/gen-docs/kyma_alpha_add_module.md:56]:
```suggestion
* [kyma alpha add](kyma_alpha_add.md)	 - Adds a resource to the Kyma cluster.
```

File: docs/gen-docs/kyma_alpha_delete.md [hunk 13]
```diff
@@ -22,5 +22,5 @@ Use this command to disable a resource in the Kyma cluster.
 ## See also
 
 * [kyma alpha](kyma_alpha.md)	 - Experimental commands
-* [kyma alpha disable module](kyma_alpha_disable_module.md)	 - Disables a module in the cluster or in the given Kyma resource.
+* [kyma alpha delete module](kyma_alpha_delete_module.md)	 - Delete a module in the cluster or in the given Kyma resource.
```
Comment (mmitoraj) [inline docs/gen-docs/kyma_alpha_delete.md:25]:
```suggestion
* [kyma alpha delete module](kyma_alpha_delete_module.md)	 - Deletes a module from the cluster or from the given Kyma resource.
```

File: docs/gen-docs/kyma_alpha_delete_module.md [hunk 14]
```diff
@@ -1,12 +1,12 @@
 ---
-title: kyma alpha disable module
+title: kyma alpha delete module
 ---
 
-Disables a module in the cluster or in the given Kyma resource.
+Delete a module in the cluster or in the given Kyma resource.
```
Comment (mmitoraj) [inline docs/gen-docs/kyma_alpha_delete_module.md:5]:
```suggestion
Deletes a module from the cluster or from the given Kyma resource.
```

File: docs/gen-docs/kyma_alpha_delete_module.md [hunk 15]
```diff
@@ -51,5 +51,5 @@ Disable "my-module" from the "alpha" channel in "default-kyma" in "kyma-system"
 
 ## See also
 
-* [kyma alpha disable](kyma_alpha_disable.md)	 - Disables a resource in the Kyma cluster.
+* [kyma alpha delete](kyma_alpha_delete.md)	 - Disables a resource in the Kyma cluster.
```
Comment (mmitoraj) [inline docs/gen-docs/kyma_alpha_delete_module.md:54]:
```suggestion
* [kyma alpha delete](kyma_alpha_delete.md)	 - Deletes a resource from the Kyma cluster.
```

**Review by jeremyharisch — APPROVED** [review]

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**